iphone/Classes/FilesystemModule.m

```diff
@@ -171,16 +180,21 @@ - (JSValue *)getFile
 {
   NSArray *args = [JSContext currentArguments];
   NSString *newpath = [self pathFromComponents:args];
+  TiFile *fileProxy = [self getFileProxy:newpath];
+  return [self NativeToJSValue:fileProxy];
+}
 
-  if ([newpath hasPrefix:[self resourcesDirectory]] && ([newpath hasSuffix:@".html"] || [newpath hasSuffix:@".js"] || [newpath hasSuffix:@".css"] || [newpath hasSuffix:@".json"])) {
-    NSURL *url = [NSURL fileURLWithPath:newpath];
+- (TiFile *)getFileProxy:(NSString *)path
+{
+  if ([path hasPrefix:[self resourcesDirectory]] && ([path hasSuffix:@".html"] || [path hasSuffix:@".js"] || [path hasSuffix:@".css"] || [path hasSuffix:@".json"])) {
```

Comment:
I believe we can remove the suffix checks for `html`, `css` since we don't encrypt those.